src/App.jsx

```diff
@@ -1,11 +1,19 @@
 import { Route, Routes } from "react-router-dom";
 import HomePage from "./pages/HomePage";
 import "./App.css";
+import SignupPage from "./pages/AuthPage/SignupPage";
```

Comment:
Instead of doing this hard coded you can create index.js and can import here both signup and login page together

Reply:
Yes you are right